sympy/printing/mathml.py

```diff
@@ -13,12 +13,10 @@
 from .conventions import split_super_sub, requires_partial
 
 
-class MathMLPrinter(Printer):
-    """Prints an expression to the MathML markup language
+class MathMLContentPrinter(Printer):
+    """Prints an expression to the Content MathML markup language.
```

Comment:
I think it's better to create a base class which contains common codes in both printers, and make both printers inherit it.

Reply:
Also, as they are public APIs, do not change existing names or create aliases from original names to them.

Reply:
I think the name change is appropriate, but you should add a `MathMLPrinter = MathMLContentPrinter` at the bottom of the file for backwards compatibility.